content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

```diff
@@ -410,6 +410,14 @@ to hold the configuration of the NGINX server:
 
     In this example. a single generic Secret with keys `tls.key`, `tls.crt`, and `ca.crt` is used.
 
+    Optionally the gateway credential may include a [certificate revocation list(CRL)](https://datatracker.ietf.org/doc/html/rfc5280)
+    using the key `ca.crl`. So, the above example would be modified as:
+
+    {{< text bash >}}
+    $ kubectl create secret -n istio-system generic client-credential --from-file=tls.key=client.example.com.key \
+      --from-file=tls.crt=client.example.com.crt --from-file=ca.crt=example.com.crt --from-file=ca.crl=/some/path/to/your-crl.pem
+    {{< /text >}}
+
```

Comment:
How about making this a tip and putting into a boilerplate?

The tip could be something like this:

```
{{< tip >}}
Optionally, the gateway credential may include a [certificate revocation list(CRL)](https://datatracker.ietf.org/doc/html/rfc5280) using the key `ca.crl`.
If so, add another argument to the above example to provide your CRL: `--from-file=ca.crl=/some/path/to/your-crl.pem` 
{{< /tip >}}
```

Reply:
I think this makes better sense.

Reply:
@frankbu @ericvn changed according to your suggestion. Please have another look